hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/streaming/BlockNodeConnection.java

```diff
@@ -5,15 +5,15 @@
 
```

Comment:
Are these changes merged in from another feature branch? These look similar to stuff in #18450